server/cpho/jinja2/indicators/view_indicator.jinja2

```diff
@@ -6,66 +6,96 @@
   {{ bc.indicator(indicator,True) }}
 {% endblock %}
 
-
 {% block content %}
-  <div class="h2">{{ tdt("Indicator Selected: ") }} {{ indicator.name }}</div>
-  <div class="mt-4">
-
-    {% if respects_rule('can_edit_indicator', indicator) %}
-      <a href="{{ url('edit_indicator', args=[indicator.id]) }}"
-         class="btn btn-primary">{{ tdt("Edit indicator") }}</a>
-    {% endif %}
-    {% if respects_rule('can_view_indicator_data', indicator) %}
-      <a href="{{ url('indicator_scoped_changelog', args=[indicator.id]) }}"
-         class="btn btn-success">{{ tdt("View changelog for this indicator's data") }}</a>
-    {% endif %}
-  </div>
-  {% if respects_rule('can_view_indicator_data', indicator) %}
-    <div class="mt-5">
-      <div class="h3">{{ tdt("Periods with data") }}</div>
-      <table class="table table-hover">
-        <thead>
-          <tr>
-            <th>{{ tdt("Period") }}</th>
-            <th>{{ tdt("Data Count") }}</th>
-            <th width="35%">{{ tdt("Progress") }}</th>
-            <th class="text-center">{{ tdt("Actions") }}</th>
-          </tr>
-        </thead>
-        <tbody>
-          {% for period,data_count in data_counts_by_period.items() %}
-            {% if data_count > 0 %}
-              {% set submission_statuses = submission_statuses_by_period[period] %}
-              {% set progress_in_table = True %}
-              <tr href="{{ url('view_indicator_for_period', args=[indicator.id, period.pk]) }}">
-                <th>{{ period }}</th>
-                <td>{{ data_count }}</td>
-                <td>{% include 'indicators/_progress_bar.jinja2' %}</td>
-                <td class="text-center">
-                  <a class='btn btn-outline-dark'
-                     href="{{ url('view_indicator_for_period', args=[indicator.id, period.pk]) }}">{{ tdt("See/Edit data") }}</a>
-                </td>
-              </tr>
-            {% endif %}
-          {% endfor %}
-        </tbody>
-      </table>
-    </div>
-    <div class="mt-5">
-      <div class="h3">{{ tdt("Add data for new periods") }}</div>
-      <div>
-        <ul>
-          {% for period,data_count in data_counts_by_period.items() %}
-            {% if data_count == 0 %}
-              <li>
-                <a href="{{ url('view_indicator_for_period', args=[indicator.id, period.pk]) }}">{{ tdt("Add data for ") }} {{ period }}</a>
-              </li>
-            {% endif %}
-          {% endfor %}
-        </ul>
+  <div class="container">
+    <div class="row">
+      <div class="h2">{{ tdt("Indicator Selected: ") }} {{ indicator.name }}</div>
+      <div class="col-md-2 mt-4">
+        <div>
+          {% if respects_rule('can_edit_indicator', indicator) %}
+            <a href="{{ url('edit_indicator', args=[indicator.id]) }}"
+               class="btn w-100 btn-outline-dark my-1">
+              <span class="bi bi-pen"></span>
```

Comment:
On this and the other `<span class="bi...` elements, I recommend adding an `aria-hidden="true"` attribute. In general, for accessibility, purely decorative icons (which these are because they're redundant to the text inside the link/button) should be hidden from screen readers.

Further, for the bootstrap font icons, i.e. the `bi-...` classes, the screen readers will likely end up reading the icon's unicode (something like "F6B3") out loud by default, which would be confusing even if paired with additional text or a label sharing the same information that the icon is intended to communicate.

If the icon wasn't purely decorative (e.g. it is the whole button without accompanying text), then you would need to add an `aria-label` or equivalent with a screen reader specific explanation. In case you do run in to that case some time, heads up that an `aria-label` directly on a span will likely be ignored by many screen readers, as a span isn't expected to have meaning requiring a special explanation. You would still want to hide the span with the font icon in it, and you would put the label on the `button`/`a`/etc that the span is inside :+1:

Reply:
Thanks for taking the time to explain it. I'll surely keep this in mind. :)